regression/crangler/loop-invariant-01/main.c

```diff
@@ -0,0 +1,27 @@
+int foo(int *arr, int size)
```

Comment:
Random place for comment: could the commit message please follow https://cbea.ms/git-commit/, as indicated in the GitHub PR template?

Reply:
I will check it out. Thank you!